ci/kind/kind-setup.sh

```diff
@@ -420,6 +462,13 @@ EOF
     if [[ $PROXY == false ]]; then
       cmd+=" --no-proxy"
     fi
+    if [[ $FLEXIBLE_IPAM == true ]]; then
+       if [[ $IPAM_VLAN_SUBNETS == "" || $IPAM_VLAN_IDS == "" ]]; then
+         echoerr "'--ipam-vlan-subnets' and '--ipam-vlan-ids' must be specified together"
+         exit 1  
+       fi
+       cmd+=" --flexible-ipam --multicast"
```

Comment:
I don't get why it should contain `--multicast`

Reply:
I used it to run ACNPAntreaIPAMMulticast tests :https://github.com/antrea-io/antrea/pull/5974#discussion_r1646858822